css/src/components/table.scss

```diff
@@ -64,4 +67,59 @@ $table-caption-spacing: $letter-spacing-medium !default;
 			}
 		}
 	}
+
+	&.table-simple {
+		border: none;
+	}
+
+	&.table-borderless {
+		border: none;
+
+		th,
+		td {
+			border: none;
+		}
+	}
+
+	&.table-complex {
+		thead tr,
+		tfoot tr {
+			background-color: $table-header;
+		}
+
+		th,
+		td {
+			border: 1px solid $table-border-dark;
+		}
+	}
+
+	&.table-striped {
+		tbody tr:nth-of-type(even) {
+			background-color: $table-row;
+		}
+	}
+
+	&.table-sm {
+		th,
+		td {
+			padding: $table-cell-padding-compact;
+		}
+	}
+
+	&.table-lg {
+		th,
+		td {
+			padding: $table-cell-padding-loose;
+		}
+	}
+
+	&.table-width-reset {
```

Comment:
Width modifier `.width-auto { width: auto !important; }`?

Reply:
It feels like it can be just a general helper, but I don't think we have a space for listing that kind of random classes. Any thoughts on what's the best place for that? Or should I keep it inside the `tables.scss` file? If so, I think it should be called `table-width-auto`.

Reply:
I'd say -- if we really need this at all -- we can place our first width atomic in `/atomics/width.scss`. Otherwise, we can remove it entirely.

Reply:
I remember there were cases before where we needed to reset the table's width but can't find any cases right now. So skipping on adding this right now, but I think we should add it eventually.